spec.html

```diff
@@ -6878,7 +6817,7 @@ <h1>
       <h1>
         AsyncIteratorClose (
           _iteratorRecord_: unknown,
```

Comment:
We should also define `Iterator Record` somewhere, so that we can refer to it here. (Again, not necessarily in this PR.)